src/schema/v2/user.ts

```diff
@@ -539,6 +542,8 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
           return false
         },
       },
+      quiz,
+      updateQuiz: updateQuizMutation,
```

Comment:
Why add the mutation here rather than the default [mutations location](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/schema.ts#L292-L295)?

Reply:
assumed it should be added under the same parent as the query. incorrect assumption?

will move.

Reply:
Ah, good catch!